website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -69,7 +69,7 @@ A state version created through the API or CLI will only be listed in the UI if
 | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `pending`    | The initial status of a state version after it has been created without encoding the state data within the request. Pending state versions do not contain state data and will not appear in the UI. The workspace cannot be unlocked normally until the latest state version has been finalized.                                     |
 | `finalized`  | The state version has been uploaded successfully to Terraform Cloud or if the state version was created with a valid `state` attribute.                                                                                                                                                                                                              |
-| `discarded` | The state version has been discarded because it was provisional and not promoted to be the current state version of the workspace. |
+| `discarded` | The state version has been discarded because it was intermediate and not promoted to be the current state version of the workspace. |
```

Comment:
I am not sure about this change. Does it sound like proper English? Should it be "because it was an intermediate state"?

Reply:
Let's change this phrase to be more accurate and reword it at the same time.

```suggestion
| `discarded` | The state version has been discarded because it was superseded by a newer state version before it could be uploaded and processed. |
```